Makefile

```diff
@@ -127,13 +127,18 @@ module-docs: chpldoc
 # dependency so parallel make executions correctly build chpldoc first.
 	$(MAKE) module-docs-only
 
-symlink-docs:
-# Set up symlinks between `/doc/release` and `/doc/sphinx/source`
-	cd doc/sphinx && ./symlinks.py
-
-docs: module-docs symlink-docs
+doc-sphinx: module-docs
+	cd doc/sphinx && ${MAKE} symlink-docs
 	cd doc/sphinx && ./run-in-venv.bash ${MAKE} html
 
+clean-sphinx:
+	cd doc/shpinx && ${MAKE} cleanall
+
+docs: doc-sphinx
+
+clean-docs: FORCE
```

Comment:
I don't think we need these clean rules (`clean-docs` and `clean-sphinx`) at the top level.